### ISSUE TITLE
Unable to enable accessibility on Android 12

### ISSUE THREAD

**Issue by jpinto321** [Nain57/Smart-AutoClicker#200]
Hello,

I have a Lenovo P11 Pro 2nd Gen, with Android 12, and I'm unable to enable accessibility for this app.
I did some research and found that on Android 13 there's an option to "Allow restricted access", but on Android 12 there's no such option.
Is there any way to enable accessibility or?

I think it's because the app is not in the Play store, so it's blocked from being allowed accessibility controls.

**Comment by Hackergeek** [comment]
This video may help you to solve the problem：[Android 13 Accessibility Access Restricted Setting Enable or Bypass | Without Root & Computer
](https://www.youtube.com/watch?v=0bcLjpfrmHw&ab_channel=munchy)

**Comment by Nain57** [comment]
It seems like a device specific issue.
As I don't own it, i can't really help you. You might want to find support on a dedicated forum for your phone brand.
